libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -829,6 +832,21 @@ static DNNReturnType execute_model_tf(const DNNModel *model, const char *input_n
             out_frame->width = outputs[0].width;
             out_frame->height = outputs[0].height;
         }
+        break;
+    default:
```

Comment:
why default

Reply:
sorry, i did not see any 'default' in my local code, i may did not push the final code

Reply:
execute default when model->func_type is unsupported in tensorflow backend. i thought it's necessary to report error and free objects.

Reply:
ok,